docs/csharp/tour-of-csharp/types-and-variables.md

```diff
@@ -72,35 +72,35 @@ This summarizes C#’s numeric types.
     - `float`  : 32 bits, range from 1.5 × 10^−45 - 3.4 × 10^38,    7-digit precision
 	- `double` : 64 bits, range from 5.0 × 10^−324 - 1.7 × 10^308, 15-digit precision
 * Decimal
-    - `decimal : 128 bits, range is at least –7.9 × 10^−28 -  7.9 × 10^28, with at least 28-digit precision
+    - `decimal` : 128 bits, range is at least –7.9 × 10^−28 -  7.9 × 10^28, with at least 28-digit precision
     
-C# programs use ***type declarations*** to create new types. A type declaration specifies the name and the members of the new type. Five of C#’s categories of types are user-definable: class types, struct types, interface types, enum types, and delegate types.
+C# programs use *type declarations* to create new types. A type declaration specifies the name and the members of the new type. Five of C#’s categories of types are user-definable: `class` types, `struct` types, `interface` types, `enum` types, and `delegate` types.
```

Comment:
In this paragraph, I wouldn't use the `inline code` style. It's referring to the concept, not the keyword.